src/index.js

```diff
@@ -71,9 +70,8 @@ async function runExec(params, pathname, log) {
 async function run(request, context) {
   const { pathname } = new URL(request.url);
   const params = context.data;
-  params.token = request.headers.has('x-token') ? request.headers.get('x-token') : undefined;
   /* c8 ignore next */
-  params.service = request.headers.has('x-service') ? request.headers.get('x-service') : undefined;
+  params.domainkey = request.headers.has('authorization') ? request.headers.get('authorization').split(' ').pop() : params.domainkey || 'secret';
```

Comment:
this is a bit weak.... why use the split if you don't need/check the first part?

Reply:
Would you use a regex?